test/e2e/tests/ppom-blockaid-alert-metrics.spec.js

```diff
@@ -282,35 +283,21 @@ describe('Confirmation Security Alert - Blockaid @no-mmi', function () {
         await driver.clickElement('#maliciousApprovalButton');
 
         // Wait for confirmation pop-up
-        let windowHandles = await driver.waitUntilXWindowHandles(3);
-        await driver.switchToWindowWithTitle(
-          WINDOW_TITLES.Notification,
-          windowHandles,
-        );
+        await switchToNotificationWindow(driver, 3);
```

Comment:
This E2E is currently disabled. Could we extract these fixes to a separate PR?

Reply:
Oh, hah, didn't notice the `it.skip`, I guess it happened after I started!  Reverted that file for now.